swarm/src/handler/either.rs

```diff
@@ -29,6 +29,8 @@ use libp2p_core::upgrade::{EitherUpgrade, UpgradeError};
 use libp2p_core::{ConnectedPoint, Multiaddr, PeerId};
 use std::task::{Context, Poll};
 
+/// Auxiliary type to allow implementing [`IntoConnectionHandler`]. As [`IntoConnectionHandler`] is
```

Comment:
as per https://github.com/libp2p/rust-libp2p/pull/2867/files#r973874109 added a comment explaining existance of the type